README.md

```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
 
 ## Installation
 
-* `git clone <repository-url>` this repository
-* `cd @ember/optional-features`
-* `yarn install`
+```bash
+ember install @ember/optional-features
+```
 
-## Running
+## Usage
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+### List available features
+Features will only be available in canary builds of ember-source. To list all available features run:
 
-## Running Tests
+```bash
+ember feature:list
+```
 
-* `yarn test` (Runs `ember try:each` to test your addon against multiple Ember versions)
-* `ember test`
-* `ember test --server`
+### Enable/disable features
+To enable a feature, run:
 
-## Building
+```bash
+ember feature:enable some-feature
+```
 
-* `ember build`
+Similarly, if you want to disable a feature, you can run:
 
-For more information on using ember-cli, visit [https://ember-cli.com/](https://ember-cli.com/).
+```bash
+ember feature:disable some-feature
+```
+
+### Installing ember-source canary
```

Comment:
@rwjblue can you confirm this is correct

Reply:
I spoke with @rwjblue on slack about this, and currently this is the best way. We need to fix the builds app to have correct links to canary builds, and then that will be the preferred method. I created an issue to do that work and assigned to myself https://github.com/ember-learn/builds/issues/62